CHANGELOG.md

```diff
@@ -12,14 +12,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 - [2076](https://github.com/FuelLabs/fuel-core/pull/2076): Replace usages of `iter_all` with `iter_all_keys` where necessary.
 
-#### Breaking
-
+### Breaking
```

Comment:
```suggestion
#### Breaking
```

It should be `####`. It is like a subsection in the `Changed`=) Each `Add`, `Removed`, `Changed`, `Fixed` sections could have non-breaking and breaking subsecitons

Reply:
Ah okay, I thought it was a mistake. Will revert.

Reply:
Done ✅